modes.xml

```diff
@@ -35,6 +35,20 @@
         <copy-directory type="scm">@game:\data</copy-directory>
         <copy-directory type="cleo">@game:\CLEO</copy-directory>
     </mode>
+    <mode extends="vc" id="vc_mobile" title="VC Mobile" game="vc" type="">
+        <data>@sb:\data\vc_mobile\</data>
+        <opcodes>@sb:\data\vc_mobile\VCSCM.ini</opcodes>
+        <examples>@sb:\data\vc_mobile\opcodes.txt</examples>
+        <classes>@sb:\data\vc_mobile\classes.db</classes>  
+        <variables>@sb:\data\vc_mobile\CustomVariables.ini</variables>
+        <keywords>@sb:\data\vc_mobile\keywords.txt</keywords>
+        <missions>@sb:\data\vc_mobile\missions.txt</missions>        
+        <constants>@sb:\data\vc_mobile\constants.txt</constants>
+        <templates>@sb:\data\vc_mobile\templates.txt</templates>
```

Comment:
@XMDS Are those files all different in VC PC and VC Mobile ? Can we re-use something from VC PC mode and remove duplicated files?